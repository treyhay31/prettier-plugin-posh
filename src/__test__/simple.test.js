describe('simple snapshot', () => {
  it('should match the suggested output', () => {
    expect(`
      $name = "dobby"
      $sum = 20 + 80
      Write-Host "$name is $sum"
`).toMatchSnapshot()
  })
})
