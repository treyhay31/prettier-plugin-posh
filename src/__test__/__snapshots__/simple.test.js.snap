// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple snapshot should match the suggested output 1`] = `
"
      $name = \\"dobby\\"
      $sum = 20 + 80
      Write-Host \\"$name is $sum\\"
"
`;
